devices/common_patches/f7_tim.yaml

```diff
@@ -0,0 +1,52 @@
+TIM*:
+  _delete:
+    - CRR6
+  CR1:
+    _delete:
+      - UIFREMAP
+    _add:
+      UIFREMAP:
+        description: UIF status bit remapping
+        bitOffset: 11
+        bitWidth: 1
+
+TIM1[0134]:
+  _delete:
+    - SMCR
+  CNT:
+    _add:
+      UIFCPY:
+        description: >
+          UIF copy
+          This bit is a read-only copy of the UIF bit of the TIMx_ISR register.
+          If the UIFREMAP bit in the TIMxCR1 is reset, bit 31 is reserved and read at 0
+        bitOffset: 31
+        bitWidth: 1
+        access: read-only
+
+TIM3:
+  CNT:
+    _delete:
+      - CNT_H
+  ARR:
+    _delete:
+      - ARR_H
+  CCR%s:
+    _delete:
+      - CCR1_H
+
+TIM9:
+  SMCR:
+    _add:
+      SMS_3:
+        description: Slave mode selection - bit 3
+        bitOffset: 16
+        bitWidth: 1
+
+TIM[1-58]:
+  DMAR:
+    _modify:
+      # Self-inconsistency between the register map and the
+      # specific register field documentation in RMs
+      DMAB:
+        bitWidth: 32
```

Comment:
![image](https://user-images.githubusercontent.com/159235/131802443-0fb8b46a-56fc-4710-9049-d3bd8649b33e.png)
![image](https://user-images.githubusercontent.com/159235/131802526-0006c362-792a-4143-a4e5-e1aa4b318fa7.png)
![image](https://user-images.githubusercontent.com/159235/131802623-311652f6-1eba-4867-9122-49afbeb39787.png)
I'm not really sure which one to believe, nor how this register is supposed to be used.
I saw that on h7 it's using 32 bits, and since there seems to have more work for this family I thought this might be the right value to use.

Reply:
Ah, I see. I checked and at least on the F767 hardware this register is 32-bit for TIM2 were the RM says 16, which is what you would expect since it can be used to read/write any of the timer's registers, of which many are 32-bit (even on 16-bit timers). So let's leave this as 32.